v2.2/build-a-python-app-with-cockroachdb-sqlalchemy.md

```diff
@@ -54,17 +48,26 @@ Create a certificate and key for the `maxroach` user by running the following co
 $ cockroach cert create-client maxroach --certs-dir=certs --ca-key=my-safe-directory/ca.key
 ~~~
 
-## Step 3. Run the Python code
+## Step 4. Run the Python code
+
+The code below uses the [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/latest/) to map Python objects and methods to SQL operations.
```

Comment:
Just call it SQLAlchemy, not the SQLAlchemy ORM. We're not using the ORM half of SQLAlchemy here.

Reply:
Fixed.